### ISSUE TITLE
Django / (login,registration,logout)

### ISSUE THREAD

**Issue by cvam0000** [GlobalTechTalks/hackathon#25]
>create an app 'accounts'
urls :login , signup , logout 
>use usercreationform 
>form fields : Full name , username or handle , password , college , DOB .

**Comment by AniVerma17** [comment]
@dishantsethi do you want to claim this task? Can you work on this?

**Comment by dishantsethi** [comment]
@AniVerma17 Yeah i can

**Comment by AniVerma17** [comment]
> @AniVerma17 Yeah i can

@dishantsethi alright then, I'm assigning this issue to you.

**Comment by dishantsethi** [comment]
cool thanks @AniVerma17

**Comment by SameeranB** [comment]
Is this issue still open?
I would like to contribute to this

**Comment by AniVerma17** [comment]
@SameeranB yes the issue is open and up for grabs. Do check out the existing Django codebase on `develop` branch and work on the pending feature(s) required as per this issue.
